src/codegen/cfg.rs

```diff
@@ -678,7 +680,13 @@ impl ControlFlowGraph {
                 self.expr_to_string(contract, ns, l),
                 self.expr_to_string(contract, ns, r)
             ),
-            Expression::Variable(_, _, res) => format!("%{}", self.vars[res].id.name),
+            Expression::Variable(_, _, res) => {
+                if let Some(var) = self.vars.get(res) {
+                    format!("%{}", var.id.name)
+                } else {
+                    format!("%#{}", res)
```

Comment:
I added this for debugging where a variable was missing from the variable table. I think I'll change it to `%INVALID({})`, what do you think?

Reply:
I variable that is not included in the variable table might make the compiler generate invalid LLVM code. Although it is good including that for debugging, I think we should raise a warning when that happens, because it is not supposed to.

Reply:
Right now we don't do any validation on the CFG. I've added an `eprintln!()` with an error